charts/genai-toolkit/values.yaml

```diff
@@ -19,9 +19,18 @@ apiv2:
     labels: {}
   apiKeyLifetimeDays:
 
+aws:
```

Comment:
These values are only used in the api deployment but statically defined in the ui deployment

Reply:
yes we need to update the ui deployment to use these same values

Reply:
I have to rebase my PR that includes those changes off of these new charts changes and such anyways so we can just use that - I'll resolve this